### PR TITLE
Filling existing fields form while editing a view

### LINKED ISSUE #14926: View overview: add an Edit link for each view
### Is your feature request related to a problem? Please describe.

The list of views should show an edit link for each view. From the navigation pane, there's a "Views" text which can be clicked on to show all the views in the database. There's a list of tabs, "Browse," "Structure," etc — I propose adding an "Edit" link here.

I don't think we can add "Edit" to the dropdown list for working with multiple views, because it wouldn't make sense to edit more than one at once.

- phpMyAdmin version:

QA_4_8

image

### PR DESCRIPTION
**Pull request by bahl24**
### Description

Fills form based on GET params available

Fixes #14926 https://github.com/phpmyadmin/phpmyadmin/issues/14916#issuecomment-459736492
Testing-
Go to structure tab of a view and click on edit view.

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Comment by bahl24** [comment]
@ibennetch @williamdes Kindly test the pr by Testing-
Go to structure tab of a view and click on edit view.
This is in response to https://github.com/phpmyadmin/phpmyadmin/issues/14916#issuecomment-459736492

**Comment by bahl24** [comment]
@ibennetch @williamdes Both the issues #14926 #14916  have been fixed, kindly check and test.

**Comment by bahl24** [comment]
@ibennetch @williamdes Can you test this?

**Comment by bahl24** [comment]
@williamdes @ibennetch Kindly take a look at it

**Comment by bahl24** [comment]
@ibennetch What do you think?

**Comment by williamdes** [comment]
@sudo-bot Deploy pr

**Comment by williamdes** [comment]
The code looks really good, when I have tested it I will merge this PR ;)

**Comment by bahl24** [comment]
> The code looks really good, when I have tested it I will merge this PR ;)

I am glad you liked it, while testing kindly check why codecov test is failing. Or maybe I should push recent changes to the branch.

File: view_create.php [hunk 1]
```diff
@@ -170,19 +170,17 @@
     exit;
 }
 
-$sql_query = ! empty($_GET['sql_query']) ? $_GET['sql_query'] : '';
-
 // prefill values if not already filled from former submission
 $view = array(
-    'operation' => 'create',
+    'operation' => (isset($_GET['view']['operation']) ? $_GET['view']['operation'] : ''),
```
Comment (mauriciofauth) [inline view_create.php]: You can use `'operation' => $_GET['view']['operation'] ?? '',`.

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

File: view_create.php (hunk 1 above)
Comment (mauriciofauth) [reply view_create.php]: Sorry, I forgot that the null coalescing operator is PHP 7+ and this PR is targeting _QA_4_8_.

**Comment by bahl24** [comment]
Kindly test this, I think there should be no issues

**Comment by bahl24** [comment]
@mauriciofauth @williamdes Did you test this PR?

**Review by williamdes — APPROVED** [review]
Merged, thank you @bahl24 for the contribution !
I have done some minor changes and rebased the branch before merge

**Pull request merged**